js/task-edit.js

```diff
@@ -25,12 +26,11 @@ function updateEditModal(id) {
     input_title_edit.value = tasks[taskIndex].title;
     input_text_edit.value = tasks[taskIndex].text;
     document.querySelector(`input[name=priorityRadios_edit][value=${tasks[taskIndex].priority}]`).checked = true;
+}
 
-    for (let i = 0; i < color_select_edit.options.length; i++) {
-        if (color_select_edit.options[i].value === tasks[taskIndex].theme) {
-            color_select_edit.options[i].selected = true;
-            break;
-        }
-    }
-
+function resetEditForm() {
+    input_title_edit.value = "";
+    input_text_edit.value = "";
```

Comment:
use `form.reset()` if you can

Reply:
very useful, thanks) 
```suggestion
    function resetForm(form, modal) {
        form.reset();
        $(modal).modal('hide');
    }
```